### ISSUE TITLE
react-native-navigation Tab Bar Icon from Android Drawables and iOS xcassets

### ISSUE THREAD

**Issue by bbhagat** [wix/react-native-navigation#2785]
[SO Link](https://stackoverflow.com/questions/48453947/react-native-navigation-tab-bar-icon-from-android-drawables-and-ios-xcassets#)

I am using react-native-navigation as a preferred navigation approach in our react-native application. We are planning on using the hybrid approach for icons where we still add icons in xcassets and drawables.

I am seeing a specific issue with icons for tab-bar on android when loading them from drawables. My code is below which works perfectly fine for iOS:

### Steps to Reproduce / Code Snippets / Screenshots

**Tabs Definition:**

```
const Tabs = [
{
    label: 'Home',
    screen: 'HomeScreen',
    //icon: require('../../assets/images/test.png'),
    icon: { uri: 'icon_home', scale: Dimensions.get('window').scale },
    selectedIcon: { uri: 'icon_home', scale: Dimensions.get('window').scale },
    title: 'Home'
},
{
    label: 'Message',
    screen: 'MessageScreen',
    icon: { uri: 'icon_home', scale: Dimensions.get('window').scale },
    selectedIcon: { uri: 'icon_home_selected', scale: Dimensions.get('window').scale },
    title: 'Message'
}]
```

**Navigation Code:**

```
_navigateToHomeScreen = () => {
Navigation.startTabBasedApp({
  tabs: Tabs,
  tabsStyle: tabsStyle,
  appStyle: tabsStyle
 });
}
```

**Tab Styles:**

```
tabStyle = {
  tabBarBackgroundColor: Colors.DefaultWhiteColor,
  tabBarButtonColor: Colors.NavigationBarItemColor,
  tabBarLabelColor: Colors.NavigationBarItemColor,
  tabBarSelectedButtonColor: Colors.PrimaryButtonBackground,
  tabBarSelectedLabelColor: Colors.PrimaryButtonBackground,
  forceTitlesDisplay: true
}
```

**What we have tried:**

- I tried using the same image in a list view and it shows up correctly so I don't think that react-native is not able to find the image from drawable.
- I tried specifying the height/width (without scale) for the image and it still does not work.
- If I use require('') to load an image which is in JS then it works correctly but we want to use drawables. 
- I tried removing **scale** and it still doesn't work.
- For iOS, loading image from xcassets and using it in tab bar works without any issues.

We are running out of options. The final solution would be to use local images and not images from native.

---
### Environment
* React Native Navigation version: 1.1.314
* React Native version: 0.52.0
* Platform(s) (iOS, Android, or both?): Android
* Device info (Simulator/Device? OS version? Debug/Release?): Emulator: Android 5.1, Android 6.0

**Comment by bbhagat** [comment]
Hi @guyca, 

Thanks for your reply. The answer to your question is below:

> Is this a supported feature of vanilla RN apps? (RN apps with react-navigation for instance)?
**[Bhavik]** As I mentioned above, I am able to load images from drawables in Android for say <Image> component which is provided by React Native. Loading images from drawables is supported out of box by react native as mentioned [here](https://facebook.github.io/react-native/docs/images.html). I am also able to load image for tabs from xcassets in iOS which is equivalent to drawables in android. So it seems that it works on iOS and does not work on android. The only thing I haven't tried is using react-navigation because we dont want to use react-navigation in our application. 

Let me know if you need more information. 

Regards,
Bhavik

**Comment by bbhagat** [comment]
Hey @guyca ,

Is this issue fixed? I see that you close the issue but I don't see a reason for closing.

Regards,
Bhavik

**Comment by vitalyrotari** [comment]
@bbhagat Nope, is not fixed. Have same problem on Android and try to find a solution for this.

**Comment by martinduran1500** [comment]
hello!! 
some one know if this issue fixed? this post it's close and i can't find to much info to solve this problem... or if some one find other solution for this ?
